app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,88 @@
 // Specific CSS for your home-page
+
+
+.container {
+  background-size: cover;
+  background-position: center;
+  padding: 50px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.background {
+  background-color: rgb(246, 235, 222);
```

Comment:
You'll b setting the propreties font-family: 'Playfair Display', serif; and color: black; to every elements inside of this one so you can just add these propreties in here directly and remove it from everywhere else. That'll be DRYer